ui/src/clientLibraries/components/ClientCSharpOverlay.tsx

```diff
@@ -17,155 +29,48 @@ const ClientCSharpOverlay: FunctionComponent<{}> = () => {
         </a>
       </p>
       <br />
-      <h5>Installing package</h5>
+      <h5>Installing Package</h5>
```

Comment:
Use imperatives in step-by-step processes.

~Installing Package~ -> Install Package